### DISCUSSION TITLE
親耳聽到也不是真的

### DISCUSSION THREAD

**Discussion by GuanYann** [1101-GS4719-C/Python-Community-GS4719-C#1247] [General]
這種技術已經被不法分子用來進行詐騙，他們可以使用 AI 模型複製他人的聲音，並假扮成對方進行詐騙行為。為了避免成為這種詐騙的受害者，我們應該保持警覺，不要輕易相信來自陌生人的語音信息，並且在進行敏感操作（如轉賬、提供個人信息等）前，最好確認該信息的真實性。
1.撥電話確認
假設接到這種詐騙電話，若想確認來電者是否偽造親人電話號碼，最快速就是掛斷打給親人確認，罪犯不太可能直接攔截你打電話。

2.問對方有困難度的問題
先確定這問題答案只有親人知道，千萬不要問在社群媒體或網路就找得到答案的問題。

3.不要接聽不明來源電話
美國 FCC 建議，不要接聽未知來電，應直接轉入語音信箱再聽；但現在很多詐騙來電都會偽裝成親人來電。

4.不要在網路公開個資
如果可以，永遠不要在社群媒體、網路公開電話號碼，也鼓勵親人不要在任何地方公開電話號碼。

https://technews.tw/2023/04/18/ai-voice-cloning-scam/

**Comment by Hsucuchia** [comment]
隨著 AI 技術的不斷發展，詐騙分子也開始利用 AI 模型來進行詐騙行為，其中之一就是模擬他人的聲音進行詐騙。在面對這種情況時，我們需要保持警覺，不要輕易相信來自陌生人的語音信息。此外，我們也可以通過撥電話確認、問對方有困難度的問題等方式來避免成為詐騙的受害者。最重要的是，我們應該避免在網路上公開個人敏感信息，以減少被詐騙的風險。

**Comment by brian431** [comment]
隨著科技的進步，新興的犯罪手法也逐漸浮現。就連以前詐騙電話是真人來模仿別人的聲音都會有人被騙了，使用AI模型錄製聲音只會讓詐騙電話更難辨識。平時大家就應該保持警覺，以免成為詐騙行為下的受害者。

**Comment by zilinng** [comment]
現在詐騙手法真的愈來愈先進，真的太可怕了，我的家人也有接過類似的詐騙電話，據他們描述聲音真的很難分辨是真是假，雖然我不確定他們聽到的是不是AI模型產出的聲音，但這提醒我們接到類似的電話時不要太緊張以至於隨便答應對方的要求，應該先冷靜的撥電話確認，不要落入詐騙集團的陷阱。

**Comment by juliaouo** [comment]
雖然目前 AI 的技術已經發展到可以模擬他人的聲音，但我認為要利用它來詐騙還是有點難度的，因為首先罪犯必須了解你的人際關係，再來他們必須取得你的親人的聲音樣本來進行模擬複製。但不可否認的還是會有被詐騙的可能，我們還是應提高警覺防止個資洩漏。

**Comment by KevinCheng42** [comment]
AI技術是一把雙面刃，有其優點及缺點，我覺得執政及立法者需要迅速建立一套法律以應對透過AI犯罪的詐騙集團，避免個資及財富的流失。

**Comment by curtis1017** [comment]
我覺得AI這個技術，要往正方面去發展，不能用在這種傷害社會的地方上，所以我覺得未來我們可能要教AI如何不要學壞，設置一些錯誤程式，不讓這些技術落入壞人之中。

**Comment by sjh501** [comment]
以目前的技術要利用AI合成人聲還是有一定的難度，但透過類似語音助理的方式擁有眾多的文字和句型樣本，再結合上自然語言生成的模型如ChatGPT等，要能夠產出擬真的語音訊息早已不是困難的事情，甚至可能會成為詐騙的趨勢，真的要小心為上。

**Comment by Jack-1D** [comment]
在我看來，當我們越依賴網路，我們的個人資料與隱私越容易暴露在危險當中。對於一般百姓實在很難將資安的知識普及化，政府反而應該要求各軟體提供商保護好使用者的個資，免於外洩。

**Comment by llh26** [comment]
現在的詐騙手法真的越來越多，甚至假裝成銀行專員，問分行名稱、電話、地址都可以直接說出來，要找經理也可以直接找人來支援，真的太恐怖了，跟各位分享一個銀行蝦皮詐騙的影片（前陣子很多人關注），大家真的要小心，最近行動支付的詐騙案也層出不窮，我爸就有遇到。我覺得可以先對自己做過的事情或買過的東西要有印象，這樣真的可以過濾一些被詐騙的機會！

https://youtu.be/hQpHT_n-oVY
